Examples/Filtering/SpatialObjectToImage1.cxx

```diff
@@ -204,12 +206,14 @@ int main( int argc, char *argv[] )
   tubePointList.clear();
   point[0] =  size[0] * 0.2 * spacing[0];
   point[1] =  size[1] * 0.8 * spacing[1];
+  point[2] =  size[2] * 0.2 * spacing[2];
   tubePoint.SetPositionInObjectSpace( point );
   tubePoint.SetRadiusInObjectSpace( size[0] * 0.05 * spacing[0] );
   tubePointList.push_back( tubePoint );
 
   point[0] =  size[0] * 0.8 * spacing[0];
   point[1] =  size[1] * 0.8 * spacing[1];
+  point[2] =  size[2] * 0.8 * spacing[1];
```

Comment:
@blowekamp Just checking... shouldn't that be `spacing[2]`? (Not sure if it matters, it's just an example, right?)

Reply:
All of us missed it 😄